core/src/test/scala/integration/kafka/zk/ZkMigrationIntegrationTest.scala

```diff
@@ -271,6 +273,9 @@ class ZkMigrationIntegrationTest {
       assertEquals(10, image.topics().getTopic("test-topic-3").partitions().size())
 
       val clientQuotas = image.clientQuotas().entities()
+      val errorUserEntity = new ClientQuotaEntity(Map("user" -> Sanitizer.sanitize(userName)).asJava)
+      assertEquals(false, clientQuotas.containsKey(errorUserEntity))
```

Comment:
This should not be needed.

Reply:
Removed it. Added `<default>` cases for user/client/ip and sanitized cases for user/client. Thanks for the review!